sklearn/preprocessing/_label.py

```diff
@@ -541,7 +543,9 @@ def _more_tags(self):
         return {'X_types': ['1dlabels']}
 
 
-def label_binarize(y, classes, neg_label=0, pos_label=1, sparse_output=False):
+@_deprecate_positional_args
+def label_binarize(y, classes, *, neg_label=0, pos_label=1,
```

Comment:
should we be consistent with `classes` here and `MultiLabelBinarizer`?

Reply:
Went with making `classes` positional for now (since it is a smaller diff). But I am okay with either.

Reply:
Hmm I think I prefer using `classes` as a keyword only option. Lets see what it looks like.